src/components/MovieInfo/MovieInfo.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import './movieInfo.css'
+
+export default function MovieInfo(props) {
+  let hr = parseInt(props.poster.runtime / 60)
+  let m = props.poster.runtime % 60
+  m = m < 10 ? '0' + m : m
+  let splittedDate = props.poster.release_date.split('-')
+  let date = [...splittedDate]
+  { [date[0], date[1], date[2]] = [date[1], date[2], date[0]] }
+  let genres =  props.poster.genres.join(' | ')
```

Comment:
Wow, this is pretty complex, nice work!

In the future when we refactor, we might want to pull this into a separate function if possible to separate the main component up a bit.